modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/CardSubscription/index.js

```diff
@@ -0,0 +1,78 @@
+import {useModal} from '@clayui/modal';
+import {useState} from 'react';
+import ModalCardSubscription from '../ModalCardSubscription';
+
+const CardSubscription = ({cardSubscriptionData}) => {
+	const [visible, setVisible] = useState(false);
+	const {observer, onClose} = useModal({
+		onClose: () => setVisible(false),
+	});
+
+	const parseAccountSubscriptionTerms = (tagName) => {
+		return tagName.toLowerCase().replace(' ', '-');
+	};
+
+	const accountSubscriptionERC = `${
+		cardSubscriptionData.accountSubscriptionGroupERC
+	}_${parseAccountSubscriptionTerms(cardSubscriptionData.name)}`;
+
+	const parseDate = (rawDate) => {
```

Comment:
This implementation was already made in `src/routes/customer-portal/components/ProjectCard/index.js`. Maybe should be good pass this to a util.js in common directory and use it both in `ProjectCard` and `CardSubscription`

Reply:
Fixed, changes are on commit **656f3c5**, _LPS-140164 Implemented date formatter functions_.